### ISSUE TITLE
Problem in postTransformer.js

### ISSUE THREAD

**Issue by mishasmaga** [docascode/type2docfx#28]
I try convert json file to yml and process faild with error:
```
C:\**\node_modules\type2docfx\dist\postTransformer.js:74
                references_1.forEach(function (ref) {
                             ^

TypeError: Cannot read property 'forEach' of undefined
    at C:\**\node_modules\type2docfx\dist\postTransformer.js:74:30
    at Array.forEach (<anonymous>)
    at C:\**\node_modules\type2docfx\dist\postTransformer.js:72:22
    at Array.forEach (<anonymous>)
    at Object.insertClassReferenceForModule (C:\**\node_modules\type2docfx\dist\postTransformer.js:68:21)
    at Object.<anonymous> (C:\**\node_modules\type2docfx\dist\main.js:94:23)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
``` 
How I can do for fix this problem?

**Comment by mishasmaga** [comment]
I fix my problem in #30 , but PR is not approved(

**Comment by qinezh** [comment]
Closed as the issue has already been fixed in latest version.
